app/components/Form/Form.js

```diff
@@ -4,6 +4,7 @@ import { Modal } from 'components/UI'
 import Pay from 'containers/Pay'
 import Request from 'containers/Request'
 import Channels from 'containers/Channels'
+import ChannelCreate from 'containers/Channels/ChannelCreate'
```

Comment:
This is probably outside of the scope of this PR, but `Form` keeps extending with new containers and is effectively a container itself.

Reply:
Yes, this is very true but I don't want to address here. This so called `Form` component could probably be more like a `Modal` container that is used for the display of all modals. The name `Form` is misleading as the use case for this container is not really about displaying forms, but more about displaying stuff on top of the main app.